app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.errors import (VaccineError, NotWearingMaskError)
+from app.cafe import Cafe
```

Comment:
It's recommended to use one style of quotes in your code. Double quotes are preferable. Please change single quotes to double quotes.